infra/modules/worklytics-connector-specs/main.tf

```diff
@@ -275,9 +275,9 @@ EOT
       target_host : "api.github.com"
       source_auth_strategy : "oauth2_refresh_token"
       secured_variables : [
-        { name : "PRIVATE_KEY", writable : true },
-        { name : "REFRESH_TOKEN", writable : true },
-        { name : "OAUTH_REFRESH_TOKEN", writable : true, lockable : true },
+        { name : "ACCESS_TOKEN", writable : true }, # writable, as needs to be shared
```

Comment:
- add ACCESS_TOKEN back
- remove REFRESH_TOKEN, GH doesn't have refresh tokens, the JWT built with private key + client id acts as that and returns back the access token (that doesn't have a refresh token)